### PR TITLE
docs(navigation-secondary): correcting issues

### LINKED ISSUE #1010: [docs] Navigation secondary
Moving comments on a [merged PR](https://github.com/RedHat-UX/red-hat-design-system/pull/994#issuecomment-1569129942) to a new issue.

@zeroedin Some notes:

### Overview tab
- `Sample component` header, image needs to be in white container with gray border (denoted in Google doc, the file name is inside a one-cell table with a white background and gray border)
- Repo status table not showing

### Style tab
- `Using the expandable menu` header, table broken
- `Small breakpoints` heading, image looks blurry, do I need to export another?
- Spacer images in table are still 2x

### Guidelines
- `Responsive design` header, I re-exported the `Slot 2 visible` and `Slot 2 hidden` subheader images because they included text which they should not have (they are on Dropbox now)

### PR DESCRIPTION
**Pull request by zeroedin**
## What I did

1. Moved sample component into a example shortcode container.
2. Fixed expandable menu table.
3. Corrected spacer image sizes.
4. Updated responsive slot 2 images.
5. Add component status table
6. Set max-width css prop on small breakpoint image to ensure no blurriness.

Closes #1010 

## Testing Instructions

1. View [deploy preview](https://deploy-preview-1011--red-hat-design-system.netlify.app/elements/navigation-secondary/)

### CODE REVIEW

**Comment by coreyvickery** [comment]
@zeroedin I tagged you in the `Navigation (secondary)` content doc. I changed three images that need to be re-uploaded, they are in Dropbox.

I also re-exported the `nav-secondary-space-breakpoints-small.png` image (the image I thought was blurry). It is `984px` as opposed to `1000px` (like the one above it), so please make sure it is not being stretched slightly because that would cause blurring.

**Comment by coreyvickery** [comment]
@zeroedin Sorry buddy, more edits.

### Global
- Remove all traces of `Subnav` links, we are delaying that page because it is used so rarely

### Overview page
- Under the `Overview` h2, change `Primary navigation` text to `primary navigation`
- Looks like there is a duplicate `Repo status` section
- Under the `Repo status` h2, in the table, change `Navigation Secondary` text to `Navigation (secondary)`

### Style page
- Under the `Slot text labels` h3, in the alert, change the `Call to Action` text to `Call to action` and the `documentation` text to `page` (keep the period at the end)
- Under the `Small breakpoints` h3, remove the `Breakpoint Spacing tokens` h3 text
- Under the `Focus` h3, in the alert, remove the extra period

### Guidelines page
- Under the `Behavior` h2, remove the italics
- Under the `Slot 2 hidden` h3, the image looks stretched or blurry
- Under the `Breakpoints` h3, in the table, add rem values below px values

**Comment by zeroedin** [comment]
### Overview page
- [ ]  Under the `Overview` h2, change `Primary navigation` text to `primary navigation`
- [ ]  Looks like there is a duplicate `Repo status` section
- [ ] Under the `Repo status` h2, in the table, change `Navigation Secondary` text to `Navigation (secondary)`

 ### Style page

- [ ] Under the `Slot text labels` h3, in the alert, change the `Call to Action` text to `Call to action` and the `documentation` text to `page` (keep the period at the end)
- [ ]  Under the `Small breakpoints` h3, remove the `Breakpoint Spacing tokens` h3 text
- [ ] Under the `Focus` h3, in the alert, remove the extra period

### Guidelines page

- [ ]  Under the `Behavior` h2, remove the italics
- [ ] Under the `Slot 2 hidden` h3, the image looks stretched or blurry


> Under the `Breakpoints` h3, in the table, add rem values below px values

We actually don't have rem values for spacers.  Our spacers values are all in px, IMHO It would be incorrect to add these.  While 16px does = 1 rem at default browser font size settings this no longer true if the user changes their local settings.  Maybe this is a hill I die on? 💀 😆

File: elements/rh-navigation-secondary/docs/10-style.md [hunk 1]
```diff
@@ -100,7 +100,7 @@
 
 
   {% alert state="info", title="Helpful Tip" %}
-    Slot 3 usually includes a call to action. To see call to action styles, visit the <a href="../call-to-action">Call to Action</a> documentation.
+    Slot 3 usually includes a Call to action. To see Call to action styles, visit the <a href="../call-to-action">Call to action</a> page.
```
Comment (nikkimk) [inline elements/rh-navigation-secondary/docs/10-style.md:103]: Would we want to do this as `[Call to action](/elements/call-to-action)` instead?

File: elements/rh-navigation-secondary/docs/10-style.md [hunk 2]
```diff
@@ -134,7 +134,7 @@
 
   ### Small breakpoints  
 
-  ![Image of secondary navigation spacing values on small breakpoints][nav-secondary-space-breakpoints-small] {style="margin-block: var(--rh-spacer-2xl, 32px);"}
+  ![Image of secondary navigation spacing values on small breakpoints][nav-secondary-space-breakpoints-small] {style="--inline-img-max-width: 984px; margin-block: var(--rh-spacer-2xl, 32px);"}
```
[the remark this answers is not in the document]
Comment (nikkimk) [reply elements/rh-navigation-secondary/docs/10-style.md:137]: @zeroedin did you want to make those changes before I approve? Or should I just go ahead and approve it?

[the remark this answers is not in the document]
Comment (zeroedin) [reply elements/rh-navigation-secondary/docs/10-style.md]: @nikkimk changes have been made

**Review by nikkimk — APPROVED** [review]
LGTM

**Pull request merged**